libportability-gfx/src/impls.rs

```diff
@@ -1971,13 +1971,23 @@ pub unsafe extern "C" fn gfxDestroyPipelineCache(
 }
 #[inline]
 pub unsafe extern "C" fn gfxGetPipelineCacheData(
-    _gpu: VkDevice,
-    _pipelineCache: VkPipelineCache,
+    gpu: VkDevice,
+    pipelineCache: VkPipelineCache,
     pDataSize: *mut usize,
-    _pData: *mut c_void,
+    pData: *mut c_void,
 ) -> VkResult {
-    //TODO: save
-    *pDataSize = 0;
+    let data = match gpu.device.get_pipeline_cache_data(&pipelineCache) {
```

Comment:
nit: could have `.map_err(map_oom)?` instead of the match

Reply:
Sorry, missed this earlier. `?` doesn't work because `VkResult` is not a `Result`.